### DISCUSSION TITLE
Directional audio from speakers

### DISCUSSION THREAD

**Discussion by supercrafter100** [Mindgamesnl/OpenAudioMc#354] [Ideas]
I'm not sure how possible this is due to the limitations of both minecraft and the audio processing. But having something like directional audio based on the position and direction of the player in-game would be a really cool addition. This would mean you could use sounds to get a player their attention towards a certain location by using the location of the speaker.

**Comment by Mindgamesnl** [comment]
Heya!

the way you describe it makes it sound exactly like how speakers already work.
have you not tried them before, or are you describing alternative methods?

Both speakers and voice chat already have spatial (stereo) audio which pans based on the player rotation compared to the audio origin

**Comment by supercrafter100** [comment]
> have you not tried them before, or are you describing alternative methods?

I only played on servers using this plugin and never noticed it. I assume they just aren't properly using it then. Thanks for the quick response!
